.gitlab/pkg_metrics.yml

```diff
@@ -51,17 +53,26 @@ send_pkg_size-a6:
   script:
     - source /root/.bashrc && conda activate ddpy3
     - mkdir -p /tmp/deb/agent
+    - mkdir -p /tmp/deb/arm-agent
     - mkdir -p /tmp/rpm/agent
+    - mkdir -p /tmp/rpm/arm-agent
     - mkdir -p /tmp/suse/agent
 
     # we silence dpkg and cpio output so we don't exceed gitlab log limit
 
     # debian
     - dpkg -x $OMNIBUS_PACKAGE_DIR/datadog-agent_6*_amd64.deb /tmp/deb/agent > /dev/null
     - DEB_AGENT_SIZE=$(du -sB1 /tmp/deb/agent | sed 's/\([0-9]\+\).\+/\1/')
```

Comment:
Similar to the directories, let's make the variable naming consistent. Let's use e.g. `AMD64_DEB_AGENT_SIZE` etc.